2023-01-08 governance polls (#2519)
makerdao/community: governance/polls/Update the wstETH oracle to assume a 1:1 stETH:ETH peg - January 8, 2024.md

---

> title: Update the wstETH oracle to assume a 1:1 stETH:ETH peg - January 08, 2024

Reviewer: ```suggestion
title: SparkLend Ethereum - Update the wstETH oracle to assume a 1 to 1 stETH to ETH peg - January 08, 2024
```

":" character is banned, please rename the file. See the poll checklists https://forum.makerdao.com/t/govalpha-handover-documentation-index/21365

Author: Ah, yes. I did remember that at some point, but forgot to make the change!

summary: Signal your support or opposition to updating the wstETH oracle to assume a 1:1 stETH:ETH peg
discussion_link: https://forum.makerdao.com/t/spark-spell-proposed-changes/23298
parameters:
    input_format: single-choice
    victory_conditions:
        - { type : plurality }
    result_display: single-vote-breakdown
version: v2.0.0
options:
   0: Abstain
   1: Yes
   2: No
start_date: 2024-01-08T16:00:00
end_date: 2024-01-11T16:00:00
---

> # Poll: Update the wstETH oracle to assume a 1:1 stETH:ETH peg - January 8, 2024

Reviewer: ```suggestion
# Poll: SparkLend Ethereum - Update the wstETH oracle to assume a 1 to 1 stETH to ETH peg - January 08, 2024
```

> The Governance Facilitators have placed a Governance Poll into the voting system on behalf of Spark Protocol and the Stability Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday,January 8 at 16:00 UTC.

Reviewer: ```suggestion
The Governance Facilitators have placed a Governance Poll into the voting system on behalf of Spark Protocol and the Stability Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, January 8 at 16:00 UTC.
```
